code/game/objects/items/gift.dm

```diff
@@ -8,13 +8,14 @@
  * Gifts
  */
 
-/var/static/blacklisted_items = typecacheof(list(
+GLOBAL_LIST_INIT(blacklisted_items, list(
 	/obj/effect,
 	/obj/belly,
 	/obj/mafia_game_board,
 	/obj/docking_port,
 	/obj/shapeshift_holder,
-	/obj/screen))
+	/obj/screen,
+	))
```

Comment:
```suggestion
	/obj/screen
	)))
```

Reply:
I also recommend renaming the list to something like gift_item_blacklist so it doesn't confuse people / immediately conflict with the next person to name their global list blacklisted_items.

Reply:
Seems right, when i get to my computer I'll change it.

Reply:
I just think that the third parenthesis is wrong.

Reply:
Weeell yes, but not if you readd the typecacheof in front of the list, because otherwise it'd be able to spawn any subtype of the blacklisted items